spec/index.md

```diff
@@ -504,9 +502,7 @@ Whilst strictly not an entity, the concept of an "account" from which money is s
 
 ### Location
 
-There is a frequent desire to label items with location, usually by attaching geographic codes for a region or area. This allows the spending or revenue to  be analysed by region or area. This geographic information can be introduced directly by classifying the item with a code, or, more frequently indirectly by associating a geographic code to e.g. an entity. For example, by labelling a supplier with their location one can then associate a spend with that supplier as spending in that location.
```

Comment:
I think the distinction is not worth making at least at this point in the development of the spec. My point in these PRs is to simplify how we specify location (that is, have just one way).  And then, once we're satisfied and deem it necessary, we can clearly define how to specify these higher-level mechanisms (e.g. associating a `location` dimension with some other dimension).